libcloud/common/dimensiondata.py

```diff
@@ -553,3 +553,64 @@ def __repr__(self):
                  'status=%s, ip=%s>')
                 % (self.id, self.name,
                    self.status, self.ip))
+
+
+class DimensionDataDefaultHealthMonitor(object):
+    """
+    A default health monitor for a VIP (node, pool or listener)
+    """
+    def __init__(self, id, name, node_compatible, pool_compatible):
+        self.id = id
+        self.name = name
+        self.node_compatible = node_compatible
+        self.pool_compatible = pool_compatible
+
+    def __repr__(self):
+        return (('<DimensionDataDefaultHealthMonitor: id=%s, name=%s>')
+                % (self.id, self.name))
+
+
+class DimensionDataPersistenceProfile(object):
+    """
+    Each Persistence Profile declares the combination of Virtual Listener
+    type and protocol with which it is
+    compatible and whether or not it is compatible as a
+    Fallback Persistence Profile.
+    """
+    def __init__(self, id, name, compatible_listeners, fallback_compatible):
+        self.id = id
+        self.name = name
+        self.compatible_listeners = compatible_listeners
+        self.fallback_compatible = fallback_compatible
+
+    def __repr__(self):
+        return (('<DimensionDataPersistenceProfile: id=%s, name=%s>')
+                % (self.id, self.name))
+
+
+class DimensionDataDefaultiRule(object):
+    """
+    A default iRule for a network domain, can be applied to a listener
```

Comment:
iRule is the feature name. Can't remember what it stands for! It's a feature of the F5 

Sent from Outlook




On Sat, Dec 5, 2015 at 8:06 PM -0800, "Tomaz Muraus" <notifications@github.com> wrote:












In libcloud/common/dimensiondata.py:
> +    Fallback Persistence Profile.
> +    """
> +    def __init__(self, id, name, compatible_listeners, fallback_compatible):
> +        self.id = id
> +        self.name = name
> +        self.compatible_listeners = compatible_listeners
> +        self.fallback_compatible = fallback_compatible
> +
> +    def __repr__(self):
> +        return (('<DimensionDataPersistenceProfile: id=%s, name=%s>')
> +                % (self.id, self.name))
> +
> +
> +class DimensionDataDefaultiRule(object):
> +    """
> +    A default iRule for a network domain, can be applied to a listener



Ah, never mind, I assume i refers to ingress?



—
Reply to this email directly or view it on GitHub.